plugins/modules/cloud/misc/proxmox.py

```diff
@@ -608,6 +610,16 @@ def main():
     timeout = module.params['timeout']
     clone = module.params['clone']
 
+    if module.params['unprivileged'] is None:
+        module.params['unprivileged'] = False
+        module.deprecate(
+            'The default value {0} for the parameter "unprivileged" is being deprecated and it will be replaced by {1}'.format(
```

Comment:
Better directly write the values here to avoid using Python formatting for them, instead of JSON formatting.
```suggestion
            'The default value `false` for the parameter "unprivileged" is deprecated and it will be replaced by `true`'
```